docs/react/getting-started.en-US.md

```diff
@@ -0,0 +1,174 @@
+
+Replace the content of `src/entries/index.js` with the following code.
+As you can see, there is no differeneces between antd's components and usual React components.
+
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { DatePicker, message } from 'antd';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: '',
+    };
+  }
+  handleChange(date) {
+    message.info('Selected Date: ' + date.toString());
+    this.setState({ date });
+  }
+  render() {
+    return (
+      <div style={{ width: 400, margin: '100px auto' }}>
+        <DatePicker onChange={value => this.handleChange(value)} />
+        <div style={{ marginTop: 20 }}>Date: {this.state.date.toString()}</div>
+      </div>
+    );
+  }
+}
+
+ReactDOM.render(<App />, document.getElementById('root'));
+```
+
+> All the components in antd are listed in the sidebar.
+
+### 4. Development & Debugging
+
+Run your project and visit http://127.0.0.1:8989
+
+```bash
+$ npm start
+```
+
+### 5. Building & Deployment
+
+```bash
+$ npm run build
+```
+
+Entry files will be built and generated in `dist` directory, then we can deploy it to different environments.
+
+> This guide is designed to help you to understand how to use antd, so it maybe not similar to what you do in real world.
+> But you can use those tools in your project, it depends on your work flow.
+
+## Compatibility
+
+Ant Design React supports all the modern browsers and IE8+.
+
+But we need to provide [es5-shim](https://facebook.github.io/react/docs/working-with-the-browser.html#browser-support) and other polyfills for IE8/9, and [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is a better choice. What's more, use [react@0.14.x](https://facebook.github.io/react/blog/2016/01/12/discontinuing-ie8-support.html) to support IE8.
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- import stylesheet -->
+    <link rel="stylesheet" href="/index.css">
+    <!-- Polyfills -->
+    <!--[if lt IE 10]>
+    <script src="https://as.alipayobjects.com/g/component/??console-polyfill/0.2.2/index.js,es5-shim/4.5.7/es5-shim.min.js,es5-shim/4.5.7/es5-sham.min.js,html5shiv/3.7.2/html5shiv.min.js,media-match/2.0.2/media.match.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+  </body>
+  <!-- import common dependencies -->
+  <script src="/common.js"></script>
+  <!-- import entry file -->
+  <script src="/index.js"></script>
+</html>
+```
+
+You may meet problems like [#28](https://github.com/ant-tool/atool-build/issues/28) and [#858](https://github.com/ant-design/ant-design/issues/858) for `babel@6.x` doesn't support IE8.
+
+[antd-init](http://github.com/ant-design/antd-init) had solved those problems, and you can refer this [webpack config](https://github.com/ant-design/antd-init/blob/f5fb9479ca973fade51fd6754e50f8b3fafbb1df/boilerplate/webpack.config.js#L4-L8)。
```

Comment:
[antd-init](http://github.com/ant-design/antd-init) had solved those problems and you can refer to this [webpack config](https://github.com/ant-design/antd-init/blob/f5fb9479ca973fade51fd6754e50f8b3fafbb1df/boilerplate/webpack.config.js#L4-L8).